src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -768,15 +768,24 @@ protected DynamoModel(IStartConfiguration config)
         /// does not already exist in the list.
         /// </summary>
         /// <param name="path"> The path to add.</param>
-        public bool AddPackagePath(string path)
+        /// <param name="file"> The file to add when importing a library.</param>
+        public bool AddPackagePath(string path, string file = "")
```

Comment:
Can we use string.Empty instead of ""? They are same but just for better readability

Reply:
I gave it a try but apparently default arguments are required to be compile-time constants, and string.Empty is not a compile-time constant, so I don't think it is possible in this case.

Reply:
Thanks for the trial, I wasn't aware of this good info before. Then LGTM